include/xsimd/arch/xsimd_neon.hpp

```diff
@@ -1380,6 +1380,40 @@ namespace xsimd
             return vcombine_f32(tmp1, tmp2);
         }
 
+        /**********
+         * insert *
+         **********/
+
+        template <class A, class T, size_t I, class = typename std::enable_if<std::is_integral<T>::value, void>::type>
+        inline batch<T, A> insert(batch<T, A> const& self, T val, index<I> pos, requires_arch<neon>) noexcept
+        {
+            switch (sizeof(T))
+            {
+            case 1:
+                return vsetq_lane_u8(val, self, I);
+            case 2:
+                return vsetq_lane_u16(val, self, I);
+            case 4:
+                return vsetq_lane_u32(val, self, I);
+            case 8:
+                return vsetq_lane_u64(val, self, I);
+            default:
+                return insert(val, self, pos, neon {});
```

Comment:
I don't understand that default o_O

Reply:
Just fixed the typo! Excess of copy-paste there.